WordPress/gutenberg
# Use a unique querystring package instead of three different ones

This PR updates our code base to use a unique `querystring` package instead of three different ones. I settled on `http-build-query` because it's the PHP compliant one.

**Testing instructions**

 - Check the term selectors
 - Check the latest posts block
 - Check the embeds
 - Check the page attributes panel (parent page)
 - Check adding links (autocompletion)
